public/app/plugins/panel/stat/suggestions.ts

```diff
@@ -29,14 +29,15 @@ export class StatSuggestionsSupplier {
       },
     });
 
-    if (dataSummary.hasStringField && dataSummary.frameCount === 1 && dataSummary.rowCountTotal < 10) {
+    // String an number field with low row count show individual rows
+    if (ds.hasStringField && ds.hasNumberField && ds.frameCount === 1 && ds.rowCountTotal < 10) {
       list.append({
         name: SuggestionName.Stat,
         options: {
           reduceOptions: {
             values: true,
             calcs: [],
-            fields: dataSummary.hasNumberField ? undefined : '/.*/',
+            fields: ds.hasNumberField ? undefined : '/.*/',
```

Comment:
```suggestion
            fields: '/.*/',
```
We are checking `ds.hasNumberField` above so it should always be truthy.